spec/models/application_spec.rb

```diff
@@ -1,5 +1,43 @@
 require 'rails_helper'
 
 RSpec.describe Application, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'data validations' do
+    it { is_expected.to validate_presence_of(:street) }
+    it { is_expected.to validate_presence_of(:phone_number) }
+    it { is_expected.to validate_presence_of(:city) }
+    it { is_expected.to validate_presence_of(:state) }
+    it { is_expected.to validate_presence_of(:zip_code) }
+    it { is_expected.to validate_presence_of(:country) }
+    it { is_expected.to validate_presence_of(:birth_place) }
+    it { is_expected.to validate_presence_of(:birth_date) }
+    it { is_expected.to validate_presence_of(:country_of_citizenship) }
+    it { is_expected.to validate_presence_of(:occupation) }
+  end
+
+  describe 'belongs_to' do
+    it { is_expected.to belong_to(:user) }
+  end
+
+  describe 'has_many/has_one' do
```

Comment:
You can do one big 'describe 'Associations' block. Then you won't need one for belongs_to and this one